### DISCUSSION TITLE
<When compiling under Ubuntu, there are a large number of Docs Errors asking for help>

### DISCUSSION THREAD

**Discussion by Mine-Langer** [LizardByte/community#613] [Sunshine] [answered]
### Select Topic Area

Question

### Body

Generating docs for page md_docs_2getting__started...
/home/langer/github.com/sunshine/docs/getting_started.md:485: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:486: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:486: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:486: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:488: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:489: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:489: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:489: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:491: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:492: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:492: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:492: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:494: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/docs/getting_started.md:495: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/third-party/doxyconfig/docs/source_code.md:141: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/third-party/doxyconfig/docs/source_code.md:141: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/third-party/doxyconfig/docs/source_code.md:142: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/third-party/doxyconfig/docs/source_code.md:156: error: Found unknown command '@ilinebr'
/home/langer/github.com/sunshine/third-party/doxyconfig/docs/source_code.md:156: error: Found unknown command '@ilinebr'
Patching output file 1/1
type lookup cache used 4532/65536 hits=24936 misses=6736
symbol lookup cache used 8191/65536 hits=35751 misses=8203
finished...
Exiting...
make[2]: *** [docs/CMakeFiles/docs.dir/build.make:71：docs/CMakeFiles/docs] 错误 1
make[1]: *** [CMakeFiles/Makefile2:430：docs/CMakeFiles/docs.dir/all] 错误 2

**Comment by ReenigneArcher** [comment]
Have you modified the docs? I don't think we're using `@ilinebr`

**Reply by Mine-Langer** [reply]
I haven't changed this document, in fact, my Sunshine application has been compiled but when it runs, the open web UI is blank and cannot be configured on the web page, so I want to get a solution

**Reply by ReenigneArcher** [reply]
`ilinebr` occurs nowhere in our docs, or anywhere in our codebase for that matter. But perhaps doxygen is converting one of the commands we use over to that. If you have an old version of doxygen you likely won't be able to build the docs. That can be disabled in cmake.

Regarding your other question, I think you have only compiled Sunshine, but not ran the install.

**Reply by Mine-Langer** [reply]
![20241231154950](https://github.com/user-attachments/assets/decad216-f08d-47f4-87af-8ee3f869c5ed)
I've compiled and built this application, and I'm running it, but when I access the https://localhost:47990 in the browser, the content of the web page is empty, and I can't enter the PIN

**Reply by ReenigneArcher** [reply]
Because you didn't run the install command. Compiling is only part of what needs to be done. Instructions are in the docs (master version if you're trying to build the master branch).

**Reply by Mine-Langer** [reply, marked as answer]
Oh, thanks, I've fixed this issue, it's my doxygen version problem, I changed 1.13 to 1.10, and when I recompile, the previous docs error disappeared, and now it's working fine

**Reply by ReenigneArcher** [reply]
This might be a doxygen-awesome-css thing then. It's only listed as compatible up to Doxygen 1.12 for now. https://github.com/jothepro/doxygen-awesome-css

**Comment by Mine-Langer** [comment]
set Doxygen 1.10
